hardware/raspberrypi/booteeprom.md

```diff
@@ -21,21 +21,25 @@ To check that the bootloader is working correctly, turn off the power, unplug ev
 
 If the Raspberry Pi is not booting it's possible that the bootloader EEPROM is corrupted. This can easily be reprogrammed using the Raspberry Pi Imager tool which is available via the [raspberrypi.org downloads page](https://www.raspberrypi.org/downloads/).
 
+Using the recovery image will erase any custom configuration options, resetting the bootloader back to factory defaults.
+
 ## Updating the bootloader
 
-We recommend setting up your Pi so that it automatically updates the bootloader: this means you will get new features and bug fixes as they are released. Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates.
+Bootloader updates are instigated during a normal 'apt update`, `apt full-upgrade' cycle, this means you will get new features and bug fixes during your normal udpates. 
+
+Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates. Although `rpi-eeprom` is available in the latest Raspberry Pi OS releases, there is usully no need to run it manually.
```

Comment:
I'm not sure if the "Although `rpi-eeprom` is available in the latest Raspberry Pi OS releases, there is usully no need to run it manually." sentence is actually necessary? :shrug: 

Although if you do decide to keep it you'll want to fix "usully" -> "usually".

Reply:
Removed,.